app/services/search/user.rb

```diff
@@ -3,10 +3,49 @@ class User < Base
     INDEX_NAME = "users_#{Rails.env}".freeze
     INDEX_ALIAS = "users_#{Rails.env}_alias".freeze
     MAPPINGS = JSON.parse(File.read("config/elasticsearch/mappings/users.json"), symbolize_names: true).freeze
+    DEFAULT_PAGE = 0
+    DEFAULT_PER_PAGE = 20
 
     class << self
+      def search_documents(params:)
+        set_query_size(params)
+        query_hash = Search::QueryBuilders::User.new(params).as_hash
+
+        results = search(body: query_hash)
+        hits = results.dig("hits", "hits").map do |user_doc|
+          prepare_doc(user_doc.dig("_source"))
+        end
+        paginate_hits(hits, params)
+      end
+
       private
 
+      def prepare_doc(hit)
+        {
```

Comment:
Right now this is the format the frontend expects so I am going to stick with it. Once everything is moved over we can work to make this cleaner and more practical